src/test/scala/cromwell/engine/store/SymbolStoreSpec.scala

```diff
@@ -0,0 +1,59 @@
+package cromwell.engine.store
+
+import cromwell.engine.store.SymbolStore.SymbolStoreKey
+import org.scalatest.{FlatSpec, Matchers}
+import cromwell.binding.{WdlExpression, WdlNamespace}
+import cromwell.binding.types.{WdlExpressionType, WdlIntegerType, WdlStringType}
+import cromwell.binding.values.{WdlValue, WdlInteger, WdlString}
+import SymbolStoreSpec._
+
+object SymbolStoreSpec {
+  val Wdl =
+    """
+      |task a {
+      |  command { echo '${message}' }
+      |  output {
+      |    String message = read_string(stdout())
+      |    Int constant = 100
+      |  }
+      |}
+      |task b {
+      |  command { echo '${message} - ${Int integer}' }
+      |}
+      |workflow wf {
+      |  call a
+      |  call b {
+      |    input: message=a.message, integer=a.constant - 75
+      |  }
+      |}
+    """.stripMargin
+
+  val Namespace = WdlNamespace.load(Wdl)
+
+  val InitialStore = Set(
+    SymbolStoreEntry(SymbolStoreKey("wf.a", "constant", None, false), WdlIntegerType, Some(WdlInteger(5))),
+    SymbolStoreEntry(SymbolStoreKey("wf.b", "integer", None, true), WdlExpressionType, Some(WdlExpression.fromString("a.constant - 75"))),
+    SymbolStoreEntry(SymbolStoreKey("wf.a", "message", None, false), WdlStringType, Some(WdlString("hello"))),
+    SymbolStoreEntry(SymbolStoreKey("wf.b", "message", None, true), WdlExpressionType, Some(WdlExpression.fromString("a.message"))),
+    SymbolStoreEntry(SymbolStoreKey("wf.a", "message", None, true), WdlStringType, Some(WdlString("hello")))
```

Comment:
Even in a test I think it would be a good idea to `input=true` or `input=false` here as otherwise that's not clear.